### ISSUE TITLE
distanceBetween function doesn't show in geolocator 6.0.0+1

### ISSUE THREAD

**Issue by hectorAguero** [Baseflow/flutter-geolocator#496]
Hi, I can't find how to call the `distanceBetween` function, even the readme example doesn't work in a sample proyect

I check the package files, and the problem is that geolocator.dart doesn't have any call to distanceBetween function of GeolocatorPlatformInterface

**Version:** 6.0.0+1

**Platform:** 
- [x] :iphone: iOS
- [x] :robot: Android

**Comment by mvanbeusekom** [comment]
Hi @hectorAguero, at the moment you can use a workaround using the following code:

``` dart
 import 'package:geolocator_platform_interface/geolocator_platform_interface.dart';

final distance = GeolocatorPlatform.distanceBetween();
```

The geolocator_platform_interface will automatically be installed with the geolocator package so it should be there. In the next update I will make sure these methods are forwarded to the geolocator plugin so you only have to import the geolocator plugin and everything works as expected.

**Comment by Arvind0515** [comment]
Thank you i too faced the same issue.

**Comment by hectorAguero** [comment]
@mvanbeusekom  solution doesn't work in 6.0.0+2, now I need to call the instance, documentation isn't updated

```dart
final distance =  GeolocatorPlatform.instance.distanceBetween();
```

I don't think this is the better aproach, because to call other geolocator functions only need to write the function name.

**Comment by mvanbeusekom** [comment]
@hectorAguero you are completely right, I have released 6.0.0+2 a bit to quickly. 

My intent was to make the `distanceBetween` and `bearingBetween` instance methods (which I did) and make global function overrides for them (so you can call them easily like all other methods, and also have support for overriding their implementation if needed). This last part I forgot to implement when I got a bit disturbed from other work coming through. 

I will release a fix (and updated documentation) for this in a few minutes. So you can easily call them like this:

``` dart
final distance = distanceBetween(56.9, 6.4, 54.3, 6.3);
```

**Comment by mvanbeusekom** [comment]
@hectorAguero, I have just released the latest version (6.0.0+3) where I implemented the changes above (b.t.w. you can still use the `GeolocatorPlatform.instance.distanceBetween` method, but there is now also a shorthand method available using simply `distanceBetween`. 

I have added unit-test and updated the documentation to make sure there will not be any confusion anymore in the future. Thanks again for reporting the issue and providing the detailed feedback.
